server/lib/routes/get-index.js

```diff
@@ -11,9 +25,20 @@ module.exports = function (config) {
   route.path = '/';
 
   route.process = function (req, res) {
+    var time = Date.now();
+    var flowId = crypto.randomBytes(16).toString('hex');
+
     res.render('index', {
+      flowBeginTime: time,
+      flowId: flowId,
       staticResourceUrl: STATIC_RESOURCE_URL
```

Comment:
Yep, it's needed. `staticResourceUrl` is added by the build step for production. Can you leave a comment to remind the next person?